File: Java/src/World/body/animal/Sheep.java
package World.body.animal;

import GUI.Color_obj;
import World.Point;
import World.World;
import World.body.Animal;
import World.body.Body;

import java.io.Serializable;

public class Sheep extends Animal implements Serializable {

    public Sheep(Point point_location, World world) {
        super(point_location, world);
        setName("Sheep");
        setColor(Color_obj.SHEEP);
        setInitiative(4);
        setPower(4);
    }

    @Override
    protected boolean repel_attack(Body attacker) {
        return false;
    }
}

File: Python/World/Body/Animal/sheep.py
from GUI.colors import Colors
from World.Body.animal import Animal
from World.point import Point


class Sheep(Animal):
    def __init__(self, point_location, world):
        super().__init__(point_location, world)
        self.set_name("Sheep")
        self.set_color(Colors.SHEEP)
        self.set_initiative(4)
        self.set_power(4)

    def repel_attack(self, attacker):
        return False
